internal/data/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
It's missing the package here @jumaallan

Reply:
We don't add/put package name on manifest anymore